### ISSUE TITLE
Fast Scanning - I am struggling with this

### ISSUE THREAD

**Issue by booyakafoo** [fhriley/kodi-headless-novnc#17]
I know this isn't a direct feature of your docker image, but you did provide some guidance on how to setup fast scanning.  I'm not sure I set this up correctly.

https://github.com/fhriley/kodi-headless-novnc?tab=readme-ov-file#fast-scanning

When adding the media source, do I add and smb:// path or the  /media/ path?  Since the from is the smb path, I think I need to use that as the source and the substitution is going to see the request for a scan of the smb:// path and replace it with the media path which is then physically scanned.  Is that correct?

I don't see anything in the logs about the substitution happing so I'm not sure if it's working or not.  The logs look like it's scanning the smb path so it's confusing to me.  Thanks

**Comment by acidzab** [comment]
> I know this isn't a direct feature of your docker image, but you did provide some guidance on how to setup fast scanning. I'm not sure I set this up correctly.
> 
> https://github.com/fhriley/kodi-headless-novnc?tab=readme-ov-file#fast-scanning
> 
> When adding the media source, do I add and smb:// path or the /media/ path? Since the from is the smb path, I think I need to use that as the source and the substitution is going to see the request for a scan of the smb:// path and replace it with the media path which is then physically scanned. Is that correct?
> 
> I don't see anything in the logs about the substitution happing so I'm not sure if it's working or not. The logs look like it's scanning the smb path so it's confusing to me. Thanks

Exactly, path substitution works like that: keeps the original path but uses the substituted one. So if you have in the first lines of your log a thing like that:

```
2024-09-12 12:30:13.767 T:36      debug <general>: Configuring path substitutions
2024-09-12 12:30:13.767 T:36      debug <general>:   Registering substitution pair:
2024-09-12 12:30:13.767 T:36      debug <general>:     From: [smb://<your samba mount>]
2024-09-12 12:30:13.767 T:36      debug <general>:     To:   [/<your substitute>/]
```

means that you've configured correctly
